gamebook/helpers.py

```diff
@@ -11,9 +12,12 @@ def trait_init():
 
     output_instance = TerminalOutput()
 
-    trait1 = gamebook.trait.Trait("Skill", 6, 1)
-    trait2 = gamebook.trait.Trait("Luck", 6, 1)
-    trait3 = gamebook.trait.Trait("Stamina", 12, 2)
+    trait1 = gamebook.trait.Trait(gamebook.important_strings.power_trait_name,

```

Comment:
why not name them like `power_trait` / `luck_trait` etc...? `1/2/3` are not understandable.